### PR TITLE
Adding Luis Rico public ssh key

### PR DESCRIPTION
**Pull request by akraitman**
Fixes: https://tracker.ceph.com/issues/51155

Signed-off-by: Adam Kraitman <akraitma@redhat.com>

### CODE REVIEW

**Comment by djgalloway** [comment]
No periods, please.  That's bound to break some automation.  Change to `luisrico.pub` here and in the secrets repo.

**Comment by akraitman** [comment]
> No periods, please. That's bound to break some automation. Change to `luisrico.pub` here and in the secrets repo.

Done

File: ssh/luisrico.pub [hunk 1]
```diff
@@ -0,0 +1 @@
+sh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC+ZvZuogU+WfyGj4BR2xYllNoQqAoxQ6AM0IYddBV2pL1xgTM/SopuAxDRP0q2stpozMtjVjlvuiIgr/caqyTuOgYtYCvB09ppefqdnCDrJJBCOKpjUEnCpAKcBMMuz9V32Zqt5FgpIi0Mu7Gp/QCvB97imxEkoUunpdX/8Ri/xz0nJO+QqoJX5wOn91jG49XnkMCLzx+MuafhT90WFO9CV1sIVREZS+X2C7ACQuhx9Qwk7R4BXt2f3M4dA0K2nLwaAnVW50M1y6Gs50Wl+ojDR1itySOxv8YpwyRlRTOzQTyVMnG0Qsaxxwp+hy4b/vQW3NIUvHYdTyvqALTQdGHT lricomor@lricomor-mac
```
Comment (djgalloway) [inline ssh/luisrico.pub:1]: Missing the beginning `s`

Comment (akraitman) [reply ssh/luisrico.pub]: oops , fixed it

**Pull request merged**